test/solidity/Facets/CalldataVerificationFacet.t.sol

```diff
@@ -51,11 +52,22 @@ contract CallVerificationFacetTest is TestBase {
             swapData
         );
 
+        bytes memory standardizedCallData = abi.encodeWithSelector(
+            StandardizedCallFacet.standardizedCall.selector,
+            callData
+        );
+
         bytes memory fullCalldata = bytes.concat(callData, "extra stuff");
         calldataVerificationFacet.extractBridgeData(fullCalldata);
         calldataVerificationFacet.extractSwapData(fullCalldata);
         calldataVerificationFacet.extractData(fullCalldata);
         calldataVerificationFacet.extractMainParameters(fullCalldata);
+
+        fullCalldata = bytes.concat(standardizedCallData, "extra stuff");
+        calldataVerificationFacet.extractBridgeData(fullCalldata);
+        calldataVerificationFacet.extractSwapData(fullCalldata);
+        calldataVerificationFacet.extractData(fullCalldata);
+        calldataVerificationFacet.extractMainParameters(fullCalldata);
```

Comment:
checkBridgeData and checkSwapData calls are missing here

Reply:
Here we only care that it doesn't blow-up with extra bytes added